src/shared/views/ChangeUserName.js

```diff
@@ -87,7 +88,7 @@ function ChangeUserName(props: Props) {
     const [updatePending, setUpdatePending] = React.useState(false);
 
     const handleConfirmButtonClick = React.useCallback(() => {
-        if ((newUserName?.length ?? 0) >= MIN_USERNAME_LENGTH) {
+        if (isValidUsername(newUserName)) {
```

Comment:
We should still check for character length. There should be a validation to check that username should have at least MIN_USERNAME_LENGTH characters.